docs/releases/1.9.txt

```diff
@@ -146,6 +146,10 @@ Internationalization
 * The :func:`django.views.i18n.javascript_catalog` view now works correctly
   if used multiple times with different configurations on the same page.
 
+* Django now allows to use locale variants supported by Gettext. These are
```

Comment:
1. "Django" is a bit redundant give these are the Django release notes. How about "You can now use...". Also, can you add details about *where* in Django these locale variants can be used?
2. "gettext" shouldn't be capitalized